configs/docusaurus_demo.json

```diff
@@ -59,4 +59,4 @@
     "415526327"
```

Comment:
I wasn't sure if this needed to be changed -- is is docusuarus specific, opposed to docusaurus-demo?

Reply:
It is docusaurus specific. You do not need to change it.